src/internal/storage/fileset/postgres_store.go

```diff
@@ -15,23 +16,26 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/internal/pacherr"
 	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
 	"github.com/pachyderm/pachyderm/v2/src/internal/require"
-	"github.com/pachyderm/pachyderm/v2/src/internal/storage/kv"
 )
 
 var _ MetadataStore = &postgresStore{}
 
 type postgresStore struct {
 	db      *pachsql.DB
-	cache   kv.GetPut
+	cache   *lru.Cache[ID, *Metadata]
 	deduper *miscutil.WorkDeduper[ID]
 }
 
 // NewPostgresStore returns a Store backed by db
 // TODO: Expose configuration for cache size?
 func NewPostgresStore(db *pachsql.DB) MetadataStore {
+	mc, err := lru.New[ID, *Metadata](100)
```

Comment:
I think the configuration option StorageMemoryCacheSize needs to go away, or be read here.

Reply:
I believe StorageMemoryCacheSize refers to the size of the chunk cache.  We don't expose a way to configure this.

Reply:
You're deleting some code that reads it here, though: https://github.com/pachyderm/pachyderm/pull/8952/files#diff-f20d955c17bf7f6b9947ddec0131271bce16608c1f09b2d5dfb18092a66d37daL50

Reply:
Nevermind, I see your fix now :)

Reply:
Yeah good catch.  I just added it back.  We configure that in `storage/config.go` now.